internal/local/app.go

```diff
@@ -265,26 +313,25 @@ func FindApp(path string) (App, bool, error) {
 	}
 
 	for i := 0; i < maxDirectoryContainSearchDepth; i++ {
-		for _, config := range allConfigFiles {
-			_, err := os.Stat(filepath.Join(wd, config.String()))
-			if err != nil {
-				if os.IsNotExist(err) {
-					continue
+		if _, err := os.Stat(filepath.Join(wd, FileMdbAppMeta.String())); err != nil {
+			if os.IsNotExist(err) {
+				// .mdb/app_meta.json may not exist, so we check whether a regular config file exists.
+				if app, proceed, err := checkForConfigFile(wd); proceed || err != nil {
```

Comment:
we should still be "checking for the config file" regardless of whether there is ./mdb/app_meta.json or not. 
We need to `LoadConfig`, regardless.